packages/grid/_modules_/grid/components/cell/GridEditInputCell.tsx

```diff
@@ -33,9 +36,15 @@ export function GridEditInputCell(props: GridCellParams & InputBaseProps) {
     setValueState(value);
   }, [value]);
 
+  useEnhancedEffect(() => {
+    if (hasFocus) {
+      inputRef.current!.focus();
```

Comment:
Are we sure that `hasFocus` is false 100% of the time on the 1st render since it's a `useLayoutEffect` ?

Reply:
I didn't understad. `hasFocus` could be false or true in the first render.

Reply:
My bad
I thought that in `useLayoutEffect` first run, the ref was not yet initialized
But I tested, and it is

Reply:
> I thought that in useLayoutEffect first run, the ref was not yet initialized

@flaviendelangle ref runs with a higher priority than use layout effects :)